fetch.js

```diff
@@ -204,6 +206,21 @@
 
         if (this._bodyBlob) {
           return Promise.resolve(this._bodyBlob)
+        } else if (this._bodyStream) {
+          var reader = this._bodyStream.getReader();
+          var chunks = [];
+          var pump = function(){
+            return reader.read().then(function(result){
+              if(!result.done){
+                chunks.push(result.value)
+                return pump()
```

Comment:
This is just for turning a stream into a blob/text/arraybuffer

We don't know in advance if they want a stream/blob/text or a arraybuffer. So I thought the best default responseType should be ms-stream or moz-chunked-arraybuffer instead of blob so we build all our Response method based on streams first